### ISSUE TITLE
System Jaskiń Pawlikowskiego - wsp. otworów oraz łączenie jaskiń

### ISSUE THREAD

**Issue by dlubom** [dlubom/Jaskiniowy-Kataster-Tatr-Zachodnich#11]
FF pisał: "Całość jest sklejona w oparciu o dość pewny ciąg pomiarowy na powierzchni
(po 3 pomiary w obie strony). Zafiksowane współrzędne otworów są określone
względem wejścia do Mylnej i jeśli będziesz to przetwarzał na jakieś
"normalne" współrzędne to musisz zachować oryginalne wzajemne położenie
otworów."

**Comment by pawczak** [comment]
Tak, tego nie wiedziałem jak zrobić muszę się skonsultować z Tobą jak Ty
byś to zrobił.

czw., 11 cze 2020, 15:29 użytkownik dlubom <notifications@github.com>
napisał:

**Comment by pawczak** [comment]
Jeszcze jedna sprawa, brakowało prefixów dla Raptawickiej i Obłazkowej -
dodałem.

Ale nie mogę wysłać, piszę mi że:

[image: image.png]

Zerkniesz czy tam wszystko ok jest w ustawieniach Repo odnośnie mojego
dostępu ?

Ja sprawdzę później jeszcze raz jakie mam dane do logowania wpisane bo może
źle zerknąłem.

>

**Comment by pawczak** [comment]
Ok, chyba musisz mnie gdzieś dodać żebym mógł bezpośrednio edytować:
[image: image.png]

sob., 13 cze 2020 o 11:10 Paweł Gogolak <pafffel@gmail.com> napisał(a):

> Jeszcze jedna sprawa, brakowało prefixów dla Raptawickiej i Obłazkowej -
> dodałem.
>
> Ale nie mogę wysłać, piszę mi że:
>
> [image: image.png]
>
> Zerkniesz czy tam wszystko ok jest w ustawieniach Repo odnośnie mojego
> dostępu ?
>
> Ja sprawdzę później jeszcze raz jakie mam dane do logowania wpisane bo
> może źle zerknąłem.
>
>>

**Comment by pawczak** [comment]
@dlubom Proszę o sprawdzenie umiejscowienia domiarów powierzchniowych oraz struktury pliku pomiarów powierzchniowych.

Dodałem TODO odnośnie daty - chyba, że znamy?

Jak rozpiąłem pomiary wychodziło pare metrów różnicy. Pewnie jest to niedokładność współrzędnych gps + błędów pomiarowych. Daj znać co myślisz, czego się w takim przypadku trzymać. Domyślam się, że jak przypiszemy koordynaty gps np z geoportalu to nam się to jeszcze bardziej rozjedzie, nie pamiętam jak nadawałem wtedy koordynaty gps, wydaje mi się, że na podstawie jakichś obliczeń opartego na ciągu pomiarowym.

Ja zostawiłem takie kordynaty jakie mieliśmy + sfixowałem końce pomiarów powierzchniowych z odpowiednimi jaskiniami Obłazkowa-Raptawicka. Dodatkowo musiałem zmienić #prefix Obłazkowej bo po podpięciu punktów nazwa punktu była zbyt długa (punkt pomiarowy powierchni przypisany do punktu otworu Obłazkowej). 

Zmian nie ma dużo,  więc łatwo będzie sprawdzić czy tak chcemy to mieć.
